css/types/round.json

```diff
@@ -13,7 +13,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "108"
             },
```

Comment:
Hi @ramiy This is behind a preference `layout.css.round.enabled` that is enabled by default in nightly from FF108. See https://github.com/mdn/content/issues/22114 section "Features to document".

So this should be done like something like this:
```
            "firefox": [
                {
                  "version_added": "preview"
                },
                {
                  "version_added": "108",
                  "flags": [
                    {
                      "type": "preference",
                      "name": "layout.css.round.enabled",
                      "value_to_set": "true"
                    }
                  ]
                }
              ],
```

Reply:
done